src/openapi-generator.ts

```diff
@@ -151,42 +170,52 @@ export class OpenAPIGenerator {
       const parameters = Object.entries(propTypes).map(([key, schema]) => {
         const innerMetadata = this.getMetadata(schema);
 
-        if (innerMetadata?.name && innerMetadata.name !== key) {
+        const innerParameterMetadata = innerMetadata?.param;
```

Comment:
We should document this somewhere in the README

Reply:
Yup, I'll add it in the Readme.md. I'll probably separate them into:
1. Defining schemas
2. Defining parameters
3. Defining routes